docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: mesheryctl system status | Meshery
+permalink: reference/mesheryctl/system/status
+type: reference
+display-title: "false"
+language: en
+lang: en
+categories: en
+list: exclude
+# image: /assets/img/platforms/brew.png
+---
+
+<!-- Copy this template to create individual doc pages for each mesheryctl commands -->
+
+<!-- Name of the command -->
+# mesheryctl system status
+
```

Comment:
Further details of the behavior of the command or a description of things like image name, how long it has been running, differences between it running on Docker and Kubernetes using different examples would be great.

Reply:
following up... any further details here?